Library/Homebrew/nightly-linux/centos7/Dockerfile

```diff
@@ -0,0 +1,54 @@
+FROM centos:7
```

Comment:
What about putting this in `.github/workflows/nightly-linux/centos7/Dockerfile` instead?